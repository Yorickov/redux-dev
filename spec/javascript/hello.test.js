import React from 'react';
import Hello from '../../app/javascript/components/Hello';

it('Hello', () => {
  const wrapper = shallow(<Hello name="Dan" />);
  expect(wrapper.render()).toMatchSnapshot();
});
