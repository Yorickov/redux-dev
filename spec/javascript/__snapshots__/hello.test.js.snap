// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello 1`] = `
<div>
  Hello Dan!
</div>
`;
